alibi_detect/utils/saving.py

```diff
@@ -960,11 +961,19 @@ def load_detector(filepath: str, **kwargs) -> Data:
         k = []
 
     # check if path exists
-    if not os.path.isdir(filepath):
+    filepath = Path(filepath)
+    if not filepath.is_dir():
         raise ValueError('{} does not exist.'.format(filepath))
 
+    # Check if dill files exist, otherwise read pickle files
+    files = list(filepath.glob('*.dill'))
+    if len(files) == 2:
```

Comment:
Where does the 2 come from?

Reply:
2 as we expect a `meta.dill` and `<model>.dill` file in the directory. Admittedly this isn't particularly robust, but I'm assuming we have tight control over what exists in a given google bucket directory. I'll change this to check if `meta.dill` and `<model>.dill` exist, otherwise check if `.pickle` files exist, otherwise raise an error.